src/main/java/in/projecteka/consentmanager/ConsentManagerConfiguration.java

```diff
@@ -128,6 +139,45 @@ public ConsentManager consentRequestService(WebClient.Builder builder,
                 new UserServiceClient(builder, userServiceProperties), consentArtefactRepository, keyPair);
     }
 
+    @Bean
+    public DestinationsConfig destinationsConfig(AmqpAdmin amqpAdmin) {
+        HashMap<String, DestinationsConfig.DestinationInfo> queues = new HashMap<>();
+        queues.put(HIU_NOTIFICATION_QUEUE, new DestinationsConfig.DestinationInfo("exchange", HIU_NOTIFICATION_QUEUE));
+
+        DestinationsConfig destinationsConfig = new DestinationsConfig(queues, null);
+        destinationsConfig.getQueues()
+                .forEach((key, destination) -> {
+                    Exchange ex = ExchangeBuilder.directExchange(
```

Comment:
These exchanges should not be created like this. 
- Can we first check if the exchange exists? I think there is a flag called auto-declare or something similar
- ideally these exchanges would be created externally.  If not handling that now, please create a tech debt
- keep in mind that in a cluster, CM grant service can be multiple, and not a single one

Reply:
I have created a card to handle all this. 
https://trello.com/c/JmHRctGg